### PR TITLE
feat: create title component

### PR DESCRIPTION
**Pull request by im-klau**
## Jira issue

JIRA issue: [FSU-64](https://klau.atlassian.net/browse/FSU-64)

## Checklist before requesting a review

- [ ] Checked which branch you would merge.
- [ ] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [ ] Performed a self-review of code (bugs, clean code rules, etc).
- [ ] Checked if added variables have unique names.
- [ ] Checked if added code/function is not a duplicate. 
- [ ] Tests have been added if possible.
- [ ] New and existing unit tests pass locally with my changes.

### CODE REVIEW

**Review by AmandaKr — APPROVED** [review]

File: packages/design-system/components/title/Title.styled.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+  &:before {
+    content: "";
+    position: absolute;
+    bottom: -6px;
+    left: 0;
+    width: 100%;
+    height: 5px;
+    clip-path: ${({ size }) =>
+      `ellipse(50% ${size === `large` ? `3px` : `2px`})`};
```
Comment (paulgrym) [inline packages/design-system/components/title/Title.styled.js:40]: Great property - nice underlining effect

File: packages/design-system/components/title/__tests__/Title.test.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { render, screen } from "../../../test-utils"
+
+import Title from "../Title"
+
+const text = "Example title"
+const badge = "Example badge"
+
+describe(`Title`, () => {
```
Comment (paulgrym) [inline packages/design-system/components/title/__tests__/Title.test.jsx:8]: Maybe we should also check whether icons will render? 🤔 Or is it unnecessary?

**Review by paulgrym — APPROVED** [review]
Great job 👏 I assume test will pass after merging with main tkanks to your latest code improvements :)

File: packages/design-system/components/title/__tests__/Title.test.jsx (hunk 2 above)
Comment (im-klau) [reply packages/design-system/components/title/__tests__/Title.test.jsx:8]: yup, that's why this PR is still a draft, not ready for review (but thanks for 2 approvals anyway 😂)
I wanted to merge the icon test PR first, and then update these tests 👍

Comment (paulgrym) [reply packages/design-system/components/title/__tests__/Title.test.jsx:9]: We trust you xD

**Review by paulgrym — APPROVED** [review]
Still great 😄

**Pull request merged**